.github/pull_request_template.md

```diff
@@ -1,15 +0,0 @@
-# Description
-
-Please summarise the changes.
-
-# Related issues
-
-Please mention any github issues addressed by this PR.
-
-# Checklist
```

Comment:
I removed this as I think that the checklist is not very helpful for this repository. Maybe we can make a new one.

Reply:
Maybe we can still keep the first two items and just remove the checklist?

Reply:
Good idea, done.

Reply:
oops, messed up the commit. Now done in [48f8627](https://github.com/CQCL/pytket-docs/pull/355/commits/48f862764917eb7876b5c380a90782b0b31b8b2b)